screens/EpisodePrediction.tsx

```diff
@@ -25,7 +27,25 @@ const ThankYouScreen: FunctionComponent = () => {
         Thank you for completing Day 1 of your Video Diary! Tomorrow you will
         complete Day 2. {"\n\n"}
         Remember, if you have questions or concerns, please contact a research
-        staff member here or by phone at 978 435 2207. {"\n\n"}
+        staff member{" "}
+        <Text
+          style={ThankYouScreenStyles.bodyLink}
+          onPress={() =>
+            // TODO: please fix this link
+            Linking.openURL("https://www.youtube.com/watch?v=dQw4w9WgXcQ")
```

Comment:
I wonder if we should store this link as a constant since we are using it in a few places.

Reply:
Should we create a new file in the Constants folder?

Reply:
Sounds great to me.